packages/legend-application-studio/src/stores/editor-state/element-editor-state/service/ServiceExecutionState.ts

```diff
@@ -373,8 +373,8 @@ export class ServicePureExecutionQueryState extends LambdaEditorState {
 }
 
 export interface ServiceExecutionContext {
-  mapping: PackageableElementReference<Mapping>;
```

Comment:
The changes in this file make me a bit nervous as they could lead to some bugs in the future. 
Could we maybe add like a new `ServiceExecutionState` called `InlineServiceExecutionState` or something to that terms and just build that when we `buildExecutionState` if there are no mapping/runtime.
And then ensure `ServiceExecutionContext` always has a defined mapping/runtime. 
Wdyt?